Reason: Always allow extra leeway in rate limiting tests,
because CI environment variable is not set on the builders.
Upstream: No

diff --git a/pkg/middlewares/ratelimiter/rate_limiter_test.go b/pkg/middlewares/ratelimiter/rate_limiter_test.go
index 26d18580..9545d5bd 100644
--- a/pkg/middlewares/ratelimiter/rate_limiter_test.go
+++ b/pkg/middlewares/ratelimiter/rate_limiter_test.go
@@ -5,7 +5,6 @@ import (
 	"fmt"
 	"net/http"
 	"net/http/httptest"
-	"os"
 	"testing"
 	"time"
 
@@ -323,9 +322,5 @@ func TestRateLimit(t *testing.T) {
 }
 
 func computeMinCount(wantCount int) int {
-	if os.Getenv("CI") != "" {
-		return wantCount * 60 / 100
-	}
-
-	return wantCount * 95 / 100
+	return wantCount * 60 / 100
 }
